src/utils/getReactNativeVersion.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright 2017-present, Callstack.
+ * All rights reserved.
+ *
+ * @flow
+ */
+const path = require('path');
+
+module.exports = function getReactNativeVersion(cwd: string) {
+  try {
+    // We deliberately want to require a dynamic string, so we have to disable flow.
+    // $FlowFixMe
+    const pak = require(path.join(
+      cwd,
+      'node_modules/react-native/package.json'
+    ));
+
+    return pak.version;
+  } catch (e) {
+    // Ignore
```

Comment:
Why should we ignore that we didn't find React Native? I would like to show an error that project is likely without react-native. @zamotany @Krizzu ?

Reply:
I think that is up to the consumer of `getReactNativeVersion`, not the function itself.

Reply:
Yeah, you are right, it will return null anyway. Which is good. 👍